recipes/blast/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "2.6.0" %}
+{% set version = "2.7.0" %}
 
 package:
     name: blast
     version: {{ version }}
 
 source:
     fn: ncbi-blast-{{ version }}+-src.tar.gz # [linux]
-    url: http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz # [linux]
-    md5: c8ce8055b10c4d774d995f88c7cc6225 # [linux]
+    url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz # [linux]
+    sha256: fc338ccbf466bb3e2e042c973d17ec600fba9adf3b1292a496ad5023ed1c759a # [linux]
     patches:
       - boost_106400.patch # [linux]
     fn: ncbi-blast-{{ version }}+-x64-macosx.tar.gz # [osx]
-    url: http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-x64-macosx.tar.gz # [osx]
-    md5: 7dc2ebc86b3a064bae6a3e378495d414 # [osx]
+    url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-x64-macosx.tar.gz # [osx]
+    sha256: c295aa858638a8194d6b4abbefedaa2759e67ee2932e3553ffbc4afa138b20ee # [osx]
 
 build:
   number: 2
```

Comment:
Reset the build number to 0.

Reply:
Thanks - I was meaning to do that but forgot once at the command line.